### PR TITLE
chore(API-2027): migrate to npm and latest supported node

### PR DESCRIPTION
**Pull request by wjohnson-truework**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,40 +5,35 @@ on:
...
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 16
 
       # pnpm stuff
```
Comment (christianbundy) [inline .github/workflows/ci.yml:20]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -5,40 +5,35 @@ on:
...
-      - name: pnpm — install
-        run: npm i pnpm@latest -g
-      - name: pnpm - config
-        run: pnpm config set store-dir $PNPM_CACHE_FOLDER
-      - name: pnpm - install
-        run: pnpm install
+      - name: Log NPM version
+        run: npm -v
```
Comment (christianbundy) [inline .github/workflows/ci.yml:22]: up to you, but IMO this is not neccessary

**Review by christianbundy — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (wjohnson-truework) [reply .github/workflows/ci.yml:20]: Good catch

File: .github/workflows/ci.yml (hunk 2 above)
Comment (wjohnson-truework) [reply .github/workflows/ci.yml:21]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -5,40 +5,34 @@ on:
...
 
-      # pnpm stuff
-      - name: pnpm — install
-        run: npm i pnpm@latest -g
-      - name: pnpm - config
-        run: pnpm config set store-dir $PNPM_CACHE_FOLDER
-      - name: pnpm - install
-        run: pnpm install
+      - name: Log NPM version
+        run: npm -v
```
Comment (christianbundy) [inline .github/workflows/ci.yml:21]:
```suggestion
```

**Pull request merged**